tests/unit/Model/Command/ForgetTest.php

```diff
@@ -0,0 +1,42 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\Model\Command;
+
+use Lmc\Matej\UnitTestCase;
+
+class ForgetTest extends UnitTestCase
```

Comment:
Test name should match "class under test name + Test", ie. ForgetUserTest or better UserForgetTest.